integration-tests/tests/circuits.rs

```diff
@@ -0,0 +1,344 @@
+#![cfg(feature = "circuits")]
+
+use bus_mapping::circuit_input_builder::BuilderClient;
+use halo2::{
+    arithmetic::FieldExt,
+    circuit::{Layouter, SimpleFloorPlanner},
+    dev::{MockProver, VerifyFailure},
+    plonk::*,
+};
+use integration_tests::{get_client, log_init, GenDataOutput};
+use lazy_static::lazy_static;
+use log::trace;
+use zkevm_circuits::evm_circuit::{
+    // param::STEP_HEIGHT,
+    table::FixedTableTag,
+    witness::{block_convert, Block, Bytecode, Rw, Transaction},
+    EvmCircuit,
+};
+use zkevm_circuits::state_circuit::StateCircuit;
+
+const STEP_HEIGHT: usize = 10;
```

Comment:
We should use the one in `param` (need to export it) to set the enough randomness in instance column. The reason is because if we set instance column full of randomness value, it complains about there is no space for blinding factors, even we already disable the feature at the underlying proof system.

Another better approach is to handle this in `EvmCircuit` to avoid this setting, will fix this asap

Reply:
I've exposed the original `STEP_HEIGHT` as pub and used that in the test for now :)

Once the `STEP_HEIGHT` is used handled in the EvmCircuit we can update this.